handlers/common.go

```diff
@@ -53,7 +53,12 @@ func GetHandlers() (map[string]revents.EventHandler, error) {
 func logRequest(f revents.EventHandler) revents.EventHandler {
 	return func(event *revents.Event, cli *client.RancherClient) error {
 		logrus.Infof("Received event: Name: %s, Event Id: %s, Resource Id: %s", event.Name, event.ID, event.ResourceID)
-		return f(event, cli)
+		startTime := time.Now()
```

Comment:
A more succinct way to do this is just put

```go
startTime := time.Now()
defer func() {
   .... log crap...
}()
```
I think it's better if we always print the timing, even on error.